src/multi_stage/endogenous_retirement.jl

```diff
@@ -22,6 +22,51 @@ function get_retirement_stage(cur_stage::Int, lifetime::Int, stage_lens::Array{I
     return Int(ret_stage)
 end
 
+function update_cumulative_min_ret!(inputs_d::Dict,t::Int,Resource_Set::String,dfGen_Name::String,RetCap::Symbol)
+
+	CumRetCap = Symbol("Cum_"*String(RetCap));
+
+	if !isempty(inputs_d[1][Resource_Set])
+		if t==1
+			inputs_d[t][dfGen_Name][!,CumRetCap] = inputs_d[t][dfGen_Name][!,RetCap];
+		else
+			inputs_d[t][dfGen_Name][!,CumRetCap] = inputs_d[t-1][dfGen_Name][!,CumRetCap] + inputs_d[t][dfGen_Name][!,RetCap];
+		end
```

Comment:
It looks like the second line here is the sum of the first line plus the term `inputs_d[t-1][dfGen_Name][!,CumRetCap]`

Could this be reformulated to 
```
A = B
if t==1
   A += C
end
```